Source/ACE/Entity/Container.cs

```diff
@@ -150,23 +134,22 @@ public void UpdateWieldedItem(Container container, uint itemId)
 
         public virtual List<KeyValuePair<ObjectGuid, WorldObject>> GetCurrentlyWieldedItems()
         {
-            return inventory.Where(wo => wo.Value.WielderId != null && wo.Value.CurrentWieldedLocation < EquipMask.WristWearLeft).ToList();
+            // return Inventory.Where(wo => wo.Value.IntProperties != null && wo.Value.CurrentWieldedLocation < EquipMask.WristWearLeft).ToList();
+            return null;
         }
 
         public virtual WorldObject GetInventoryItem(ObjectGuid objectGuid)
         {
-            var containers = inventory.Where(wo => wo.Value.ItemCapacity > 0).ToList();
+            if (Inventory.ContainsKey(objectGuid))
+                return new GenericObject(Inventory[objectGuid]);
+
+            var containers = Inventory.Where(wo => wo.Value.ItemsCapacity > 0).ToList();
```

Comment:
That was legacy from when we did not have types set and everything was a debug object, I will fix.      This was a lot more involved than I thought, so I just overlooked that. Thanks for catching it!